CHANGELOG.next.asciidoc

```diff
@@ -420,6 +420,7 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 - Add support for region/zone for Vertex AI service in GCP module {pull}41551[41551]
 - Add support for location label as an optional configuration parameter in GCP metrics metricset. {issue}41550[41550] {pull}41626[41626]
 - Added `tier_preference`, `creation_date` and `version` fields to the `elasticsearch.index` metricset. {pull}41944[41944]
+- Add `use_performance_counters` to use performance counters to collect CPU related metrics on Windows. {pull}41965[41965]
```

Comment:
You should specify which module (system) and metricsets this applies to (core, cpu)